content/src/main/content/jcr_root/apps/acs-commons/authoring/accordion/clientlibs/css/accordion.less

```diff
@@ -0,0 +1,110 @@
+
+/* Accordion Styles */
+.rte-accordion {
```

Comment:
generic style name; can it be prefixed with acs-commons; so acs-commons-rte-accordion